src/styles/zIndex.js

```diff
@@ -0,0 +1,10 @@
+export default {
+  menu: 10,
+  appBar: 11,
+  leftNav: 12,
+  overlay: 13,
+  dialog: 14,
+  layer: 20,
+  popover: 20,
+  tooltip: 21,
```

Comment:
it's better if we keep some major gaps between dialog and layer, maybe someday we want to add another items between them, the tooltip must be the top of the food chain, it has to be on top of all, 3000 would be a good way to indicate that.

about overlay. well dialog must be opened over leftnav, so it's overlay has to cover leftnav, unless we overwrite dialog's overlay in dialog's source code. which isn't so bad.